tests/lib/cli.js

```diff
@@ -82,6 +83,7 @@ describe("cli", () => {
         let fixtureDir;
         const log = {
             info: sinon.spy(),
+            warn: sinon.spy(),
```

Comment:
We should also add `log.warn.resetHistory();` here:

https://github.com/eslint/eslint/blob/642197346bf02d277c2014144537aa21ab57dc59/tests/lib/cli.js#L156-L160

Reply:
Just to check - maybe you updated this locally but didn't push the change?

Reply:
😆 You know me too well. :+1: